app/views/api/v2/smart_class_parameters/main.json.rabl

```diff
@@ -2,5 +2,5 @@ object @smart_class_parameter
 
 extends "api/v2/smart_class_parameters/base"
 
-attributes :description, :override, :parameter_type, :default_value, :use_puppet_default, :required, :validator_type, :validator_rule,
+attributes :description, :override, :parameter_type, :default_value, :hidden_value, :use_puppet_default, :required, :validator_type, :validator_rule,
```

Comment:
Is this hidden_value field actually going to contain `*****`?  I had assumed (and haven't checked the global parameter field) that it would contain true/false, i.e. the value of the `hidden_value` column.

I can't see a lot of use of sending asterisks to an API user, but we really ought to be providing them with the value of the boolean so they can toggle it on and off.